code/modules/mob/living/superior_animal/roach/life.dm

```diff
@@ -1,26 +0,0 @@
-/mob/living/superior_animal/roach/Life()
```

Comment:
Put this file back and leave it empty, it needs to be there as a place to put special life code

Also there was some, at least one roach regenerates chemicals over time. This file is the place for that, where is it now?

Reply:
It is where it always was, in the file for the special roach. Since it is a property of only that one roach.